rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -97,6 +99,17 @@ def _token_is_aave_contract(self, asset: 'Asset') -> bool:
             asset_identifier=asset.identifier,
         ) == self.counterparty
 
+    def _decode_interest(
```

Comment:
Hmmm this is not the right approach. This shoudl not be in the common decoder. It should be in the aave v3 decoder. Since it's only for aave v3. Isn't there a v3 only common class? If not you need to create one.